android/app/build.gradle

```diff
@@ -24,7 +24,7 @@ if (flutterVersionName == null) {
 
 android {
     namespace "com.hjiangsu.thunder"
-    compileSdkVersion flutter.compileSdkVersion
+    compileSdkVersion 33
```

Comment:
Is this change necessary? Just wanted to know if there was a specific reason for this!